kratos/tests/test_model_part.py

```diff
@@ -5,8 +5,8 @@
 
 import sys
 
-class TestModelPart(KratosUnittest.TestCase): 
-    
+class TestModelPart(KratosUnittest.TestCase):
+
     def setUp(self):

```

Comment:
we need tests of mpcs...as i see there is nothing added here

Reply:
Added in the latest commit.